samples/client/petstore/java/feign/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -70,7 +70,6 @@ public ApiClient(String[] authNames) {
       } else if ("bearer_test".equals(authName)) {
         auth = new HttpBearerAuth("bearer");
       } else if ("http_signature_test".equals(authName)) {
-        auth = new HttpBearerAuth("signature");
```

Comment:
doesn't seems to be correct

Reply:
@borsch this generator library (like many others) actually does not yet support `http_signature_test`.
The typo I fixed in the mustache templates led to an incorrect handling of this authentication type.

I will replace the empty else-if block with an exception if the authentication name is defined in the openapi.yaml file but this generator library is in use in order to avoid an unexpected later failure.